src/plugins/data/public/ui/query_string_input/__snapshots__/query_string_input.test.tsx.snap

```diff
@@ -148,6 +148,74 @@ exports[`QueryStringInput Should disable autoFocus on EuiFieldText when disableA
               "setIsCollapsed": [MockFunction],
               "setIsVisible": [MockFunction],
             },
+            "data": Object {
```

Comment:
This commit creates unnecessary snapshot of data service, but it is known issue and was already reported before: https://github.com/elastic/kibana/pull/51818#pullrequestreview-325066355

I'll create an issue to fix these mocks in a separate effort.

Reply:
https://github.com/elastic/kibana/issues/52181